### DISCUSSION TITLE
"Going" links are preventing Mastodon verification.

### DISCUSSION THREAD

**Discussion by ghost** [JulianPrieber/littlelink-custom#285] [Issues]
When LLC switched the link URL scheme, all the links became ./going/x. Even though the rel="me" is still on Mastodon links, they can't verify because mastodon looks for a matching URL to the profile. Is there an option to disable this feature per link?

**Comment by JulianPrieber** [comment]
@MagicLike

**Comment by MagicLike** [comment]
> When LLC switched the link URL scheme, all the links became ./going/x. Even though the rel="me" is still on Mastodon links, they can't verify because mastodon looks for a matching URL to the profile. Is there an option to disable this feature per link?

I have tried it on multiple accounts and it works perfectly fine. Did you have any negative experiences or do you just assume?

**Comment by JulianPrieber** [comment]
I'm going to convert this to a discussion.

If you want to update anything, you can do that on the new thread.
